index.html

```diff
@@ -0,0 +1,145 @@
+        <li class="menu-list"><a href="#program">Program</a></li>
+        <li class="menu-list"><a href="#join">Join</a></li>
+        <li class="menu-list"><a href="#sponsor">Sponsor</a></li>
+        <li class="menu-list"><a href="#news">News</a></li>
+        <li class="campaign-menu"><a href="index.html">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main class="cultural-days-wrap" id="main">
+    <section class="invitation">
+      <h2 class="hello-text">"Hello! Sharing World"</h2>
+      <h1 class="tittle">CREATIVE COMMONS <br> GLOBAL SUMMIT <br> 2015</h1>
+      <div class="invite-wrap">
+        <p class="invite-msg">A joyfull celebration believing in the value of openness and sharing. Creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+      </div>
+      <h2 class="date-info">2015.10.15(THU) ~ 16(FRI)</h2>
+      <p class="locate">@ National Museum of Korea, Art Center Nabi and  more</p>
+    </section>
+    <section class="main-program" id="program">
+      <h2 class="secondary-tittle">Main Program</h2>
+      <div class="bar-icon"></div>
+      <div class="program-list">
+        <div class="program">
+          <img class="program-icon" src="images/program_icon_01.png" alt="">
+          <h3 class="program-tittle">Lecture</h3>
+          <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+        </div>
+        <div class="program">
+          <img class="program-icon" src="images/program_icon_02.png" alt="">
+          <h3 class="program-tittle">CC Exhibition</h3>
+          <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+        </div>
+        <div class="program">
+          <img class="program-icon" src="images/program_icon_03.png" alt="">
+          <h3 class="program-tittle">Forum</h3>
+          <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+        </div>
+        <div class="program">
+          <img class="program-icon" src="images/program_icon_04.png" alt="">
+          <h3 class="program-tittle">Workshop</h3>
+          <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+        </div>
+        <div class="program">
+          <img class="program-icon" src="images/program_icon_05.png" alt="">
+          <h3 class="program-tittle">CC Ignite</h3>
+          <p class="program-text">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Consectetur harum velit dolorem magni eos laborum.</p>
+        </div>
+      </div>
+      <div class="join-culture" id="join">
+        <a class="join-btn" href="#">Join Culture Days 2022</a>
+      </div>
+    </section>
+    <section class="featured-activities" id="news">
```

Comment:
- [OPTIONAL] RECOMMENDED I think it's better to stick to the template. Create a button `MORE` after the first two speakers as in the template. Sticking to the template is a project requirement.